skimage/measure/_marching_cubes_classic.py

```diff
@@ -33,7 +33,9 @@ def marching_cubes_classic(volume, level=None, spacing=(1., 1., 1.),
     -------
     verts : (V, 3) array
         Spatial coordinates for V unique mesh vertices. Coordinate order
-        matches input `volume` (M, N, P).
+        matches input `volume` (M, N, P). If ``allow_degenerate`` is set to
+        True, then the presence of degenerate triangles in the mesh can make
+        this array appear to have duplicate vertices.
```

Comment:
Appear to have, or have?

Reply:
"have", good catch. This is because V simply refers to the number of output vertices, and there are duplicates in the V vertices outputted. Setting `allow_degenerate` to False would reduce the `verts` list to only have unique vertices.